apps/freeablo/fagui/guimanager.cpp

```diff
@@ -189,6 +191,116 @@ namespace FAGui
                 mGoldSplitCnt = 0;
                 break;
             }
+            case FAWorld::ItemType::misc:
+            {
+                switch (item.getMiscId())
+                {
+                    case FAWorld::ItemMiscId::potionOfHealing:
+                    {
+                        FAWorld::Potion::restoreHp(mPlayer);
```

Comment:
In order to work in multiplayer, this code needs to be changed so it sends the action to the server, and then only actually performs the healing when the server sends the action back out to all clients.
The player input types are here: https://github.com/wheybags/freeablo/blob/master/apps/freeablo/faworld/playerinput.h you'll need to add a new one. They get processed here: https://github.com/wheybags/freeablo/blob/master/apps/freeablo/faworld/playerbehaviour.cpp#L70. Here's an example of sending one: https://github.com/wheybags/freeablo/blob/master/apps/freeablo/engine/localinputhandler.cpp#L29. You can send an EquipTarget as part of the event, that stores the location in inventory, and then on the ither end you just get the item from that location, make sure it's stipl there, and use it. Sorry I didn't notice before now.